app/models/async_request/job.rb

```diff
@@ -1,6 +1,41 @@
 module AsyncRequest
   class Job < ActiveRecord::Base
     serialize :params, Array
-    enum status: [:waiting, :processing, :processed]
+    enum status: { waiting: 0, processing: 1, processed: 2, failed: 3 }
+
+    def self.find_by_jwt_token(jwt_job_id)
+      find(AsyncRequest::JsonWebToken.decode(jwt_job_id)[0]['job_id'])
```

Comment:
I am not familiarize with the code so I am going to list here a couple of things that could fail and you can tell me if you took that into account:
_ `job_id` doesn't exists
_ `decode` method fails
_ `[0]` returns `nil`

Reply:
This method is no longer needed. I forgot to remove it